pages/index.js

```diff
@@ -1,17 +1,22 @@
-import dynamic from "next/dynamic";
-import {Provider as ReduxProvider} from "react-redux";
+import { Provider as ReduxProvider } from "react-redux";
 import store from "../store";
-
-const DynamicMap = dynamic(() => import("./Map"), {
-  ssr: false,
-});
+import GridLayout from "./GridLayout";
+import FileUpload from "./FileUpload";
+import React, { useState } from 'react';
 
 export default function Home() {
+  const [data, setData] = useState([]);
```

Comment:
This works for a demo. For a long-term solution, since we are already using redux, can you think about a solution to store and manage the drag-n-drop data in the Redux Store? @justin-kleid

Reply:
To clarify, this would involve using useDispatch from react redux right?

Reply:
Yes, useDispatch is one of the way to dispatch action (eg uploadfile) with payloads (eg data). You can come up with a design with the data structure that you want to save in the state and actions that update the data in the states.